src/Nix/Builtins.hs

```diff
@@ -666,7 +674,7 @@ toPath = fromValue @Path >=> toNix @Path
 pathExists_ :: MonadNix e m => m (NValue m) -> m (NValue m)
 pathExists_ path = path >>= \case
     NVPath p  -> toNix =<< pathExists p
-    NVStr s _ -> toNix =<< pathExists (Text.unpack s)
+    NVStr ns -> toNix =<< pathExists (Text.unpack (stringIntentionallyDropContext ns))
```

Comment:
`pathExists` should fail if the string has context.

Reply:
Good catch.